RELEASE/scripts/autoscend/quests/level_12.ash

```diff
@@ -825,69 +854,50 @@ boolean L12_filthworms()
 			buffMaintain($effect[Wet and Greedy], 0, 1, 1);
 		}
 		buffMaintain($effect[Frosty], 0, 1, 1);
-	}
-
-	if((!possessEquipment($item[A Light That Never Goes Out])) && (item_amount($item[Lump of Brituminous Coal]) > 0))
-	{
-		buyUpTo(1, $item[third-hand lantern]);
-		autoCraft("smith", 1, $item[Lump of Brituminous Coal], $item[third-hand lantern]);
-	}
-
-	if(possessEquipment($item[A Light That Never Goes Out]) && can_equip($item[A Light That Never Goes Out]))
-	{
-		if(weapon_hands(equipped_item($slot[weapon])) != 1)
+		
+		handleFamiliar("item");
+		handleServant("item");
```

Comment:
Unnecessary (Lash guarantees them in 1 turn regardless of item drop)

Reply:
This line only gets called if we do not have lash

Reply:
I guess it can't hurt to leave it.